content/doc/installation/from-source.en-us.md

```diff
@@ -44,7 +44,9 @@ git checkout v1.0.0
 
 ## Build
 
-Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. Depending on your requirements you possibly want to add various build tags, you can choose between these tags:
+Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. (For Windows, you can download make in [this page](http://www.equation.com/servlet/equation.cmd?fa=make)).
```

Comment:
Replace `(For Windows, you can download make in [this page](http://www.equation.com/servlet/equation.cmd?fa=make)).` with `If you are using Windows you can download make [here](http://www.equation.com/servlet/equation.cmd?fa=make).`